out/production/resources/map.json

```diff
@@ -0,0 +1,291 @@
+      "items": {
+        "basicItems": {
+          "4": {
+            "name": "poison",
+            "canBe": "get, take, drop",
+            "usedTo": ""
+          }
+        },
+        "containers": {}
+      }
+    },
+    "0,1": {
+      "name": "Basement",
+      "description": "Finally! After all of these hours and a single cup of coffee you have found Thilo. Don't forget to mention that your grade needs some revision;)",
+      "items": {
+        "basicItems": {},
+        "containers": {}
+      }
+    },
+    "2,1": {
+      "name": "Lobby",
+      "description": "When entering the building you almost slip on a doormat, weird, it usually isn't there. Everything is quieter than usual in the Lobby. The lights are off and it is, thus, darker than usual.",
+      "items": {
+        "basicItems": {
+        },
+        "containers": {
+          "5": {
+            "name": "doormat",
+            "description": "You just lifted up the doormat and turns out that there is a key underneath it.",
+            "canBe": "lift",
+            "usedTo": "",
+            "entities": {
+              "6": {
+                "name": "key",
+                "canBe": "get, take, drop",
+                "usedTo": ""
+              }
+            }
+          }
+        }
+      },
+      "obstacle" : {
+        "name": "guard",
+        "description": "The guard is blocking your entrance. You have to be clever to get in.",
+        "toNeutralize": "wallet"
+      }
+    },
+    "1,1": {
+      "name": "North Side of the Lobby",
+      "description": "There is a comfortable looking sofa and although you are tired FOCUS on your goal!! ",
+      "items": {
+        "basicItems": {
+        },
+        "containers": {
+          "7": {
+            "name": "sofa",
+            "description": "You are not strong enough - and why would you want to carry a sofa around?",
+            "entities": {
+            },
+            "canBe": "",
+            "usedTo": ""
+          }
+        }
+      }
+    },
+    "3,1": {
+      "name": "South Side of the Lobby",
+      "description": "Although the lightning is dim you can see a water bottle standing on a table in the south corner of the lobby. And of course, as the whole Amsterdam is, the VU is also dealing with pest infestation. Thinking of this, thankfully, makes your late night cravings disappear.",
+      "items": {
+        "basicItems": {
+          "8": {
+            "name": "bottle",
+            "canBe": "get, take, drop",
+            "usedTo": ""
+          },
+          "9": {
+            "name": "poison",
+            "canBe": "get, take, drop",
+            "usedTo": ""
+          }
+        },
+        "containers": {}
+      }
+    },
+    "4,1": {
+      "name": "Spar",
+      "description":  "What a temptation! A fully stocked Spar with even less security than usual, but the techno music is still on! Maybe a little bit wouldn't hurt. Maybe there is just a little bit left??",
```

Comment:
LOVE the humor and personality of the hardcoded descriptions of the rooms and spaces in your system. The code for this section of your system follows a similar logic from one room to another which can get repetitive but I appreciate the way you guys tried to make it unique. The structure is good with no glaringly obvious issues of spacing and such.